.devcontainer/base/devcontainer.json

```
@@ -1,8 +1,8 @@
 {
-  "name": "Autoware",
+  "name": "autoware:universe",
```

I think it is better to understand. 

```suggestion
  "name": "autoware:universe-devel",
```